vcr.gemspec

```diff
@@ -42,4 +42,9 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "relish"
   spec.add_development_dependency "mime-types"
   spec.add_development_dependency "sinatra"
+  ruby_version = RUBY_VERSION.split('.').map { |x| x.to_i } # TODO what about prereleases?
+  if (ruby_version <=> [3, 0, 0]) >= 0
```

Comment:
Perhaps we can check "does it start with 3"?

Reply:
I like that idea thanks, you could probably tell that I thought this would be a "quick fix" and tested things in CI instead of setting up a local testing environment. I'm going to fix that because hopefully it was just me...but 50 failing CI e-mails were quite unexpected 😆.

I'm also going to see if I can fix all the traceback noise in the builds because they're very hard to parse right now even when builds pass (which they do for most Ruby versions).